sui-execution/Cargo.toml

```diff
@@ -35,13 +35,28 @@ move-bytecode-verifier-v0 = { path = "../external-crates/move/move-execution/v0/
 move-bytecode-verifier-v1 = { path = "../external-crates/move/move-execution/v1/crates/move-bytecode-verifier" }
 move-bytecode-verifier-next-vm = { path = "../external-crates/move/move-execution/next-vm/crates/move-bytecode-verifier" }
 # move-bytecode-verifier-$CUT = { path = "../external-crates/move/move-execution/$CUT/crates/move-bytecode-verifier" }
-move-vm-runtime-latest = { path = "../external-crates/move/crates/move-vm-runtime", package = "move-vm-runtime" }
-move-vm-runtime-v0 = { path = "../external-crates/move/move-execution/v0/move-vm/runtime" }
+move-vm-runtime-latest = { path = "../external-crates/move/crates/move-vm-runtime", package = "move-vm-runtime", features = ["gas-profiler"] }
+move-vm-runtime-v0 = { path = "../external-crates/move/move-execution/v0/move-vm/runtime", features = ["gas-profiler"] }
 move-vm-runtime-v1 = { path = "../external-crates/move/move-execution/v1/crates/move-vm-runtime" }
-move-vm-runtime-next-vm = { path = "../external-crates/move/move-execution/next-vm/crates/move-vm-runtime" }
+move-vm-runtime-next-vm = { path = "../external-crates/move/move-execution/next-vm/crates/move-vm-runtime", features = ["gas-profiler"] }
 # move-vm-runtime-$CUT = { path = "../external-crates/move/move-execution/$CUT/crates/move-vm-runtime" }
```

Comment:
If it's the right move to include the `feature` in the dependency, then you will also need to update these template lines to include it.

Reply:
Alas the beauty of features, this is not the case